code/__DEFINES/flags.dm

```diff
@@ -97,6 +97,8 @@ GLOBAL_LIST_INIT(bitflags, list(1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 204
 #define NO_RUST (1<<5)
 /// Is this turf is "solid". Space and lava aren't for instance
 #define IS_SOLID (1<<6)
+/// Is this turf currently part of a map template load process
```

Comment:
is it map template loading or map loading?
It's weird that just checking for initialized doesn't work here